docs/tutorial/django.rst

```diff
@@ -328,12 +332,13 @@ Create the file ``time_app/urls.py`` with the following contents:
 .. literalinclude:: code/django/time_app_urls.py
     :language: python
 
-Open the file ``django_hello_world/urls.py`` and insert the following contents:
+Open the file ``django_hello_world/urls.py`` and replace its contents with
+the following:
 
 .. literalinclude:: code/django/urls.py
     :language: python
 
-Since you are creating a new version of your application, go back to the
+Since we are creating a new version of our application, go back to the
```

Comment:
```suggestion
Since we are creating a new version of the application, go back to the
```

IDK "our app" sounds a bit much to me. (I don't like "your app" either). I'll defer to any standards we may have though.